django-verdant/basicauth/response.py

```diff
@@ -0,0 +1,14 @@
+from django.conf import settings
+from django.http import HttpResponse
+
+
+class HttpResponseUnauthorized(HttpResponse):
+    status_code = 401
+
+    def __init__(self):
+        super(HttpResponseUnauthorized, self).__init__(
+            """<html><head><title>Basic auth required</title></head>
+               <body><h1>Authorization Required</h1></body></html>""",
+        )
```

Comment:
Check if this needs to have `content_type='text/html'` in order for this error to display correctly

Reply:
Brower seems happy with the way it is